### DISCUSSION TITLE
Request to make Dask (and Distributed) optional dependencies

### DISCUSSION THREAD

**Discussion by lifepillar** [blue-yonder/tsfresh#1060] [Ideas]
Would it be possible to ship a new release of tsfresh without the requirements in the subject? They carry several other dependencies with them, which complicates dependency management and it's wasteful if one doesn't use them.

AFAICS, Dask is already an optional requirement for tsfresh, but it is installed by default nonetheless.

**Comment by nils-braun** [comment]
Hi @lifepillar!
Thanks for your proposal, that sounds like a very good idea! Would you like to do a PR for this?

**Reply by lifepillar** [reply]
PR submitted. I hope you may tag a new release after the PR is merged.

**Reply by nils-braun** [reply]
Thanks! 0.20.2 is released
